MAPL_Base/MAPL_HistoryGridComp.F90

```diff
@@ -3099,6 +3185,19 @@ subroutine Finalize ( gc, import, export, clock, rc )
       ELSE
          if( list(n)%unit.ne.0 ) call FREE_FILE( list(n)%unit )
       END if
+      if(list(n)%monthly) then
```

Comment:
I would have thought that the alarm in the contained IF statement would account for the monthly flag as well?   Would be nice to avoid such deeply nested  conditionals.